core/tests/test_solr_index.py

```diff
@@ -51,8 +51,11 @@ def test_get_sort(self):
 
     # page_count
 
-    def test_page_count(self):
-        self.assertEqual(si.page_count(), 131)
+    # TODO the below is pulling data from my development environment
+    # commenting out until the test database is distinct
+
+    # def test_page_count(self):
+    #     self.assertEqual(si.page_count(), 131)
```

Comment:
As we discussed, this is a problem with tests sharing the same solr config.  I think I'm going to file a ticket.